src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/AvslåVedtak/AvslåVedtak.tsx

```diff
@@ -89,6 +94,7 @@ const AvslåVedtak: React.FC<{ vedtak?: AvslagBarnetilsyn }> = ({ vedtak }) => {
                 steg={Steg.BEREGNE_YTELSE}
                 nesteFane={FanePath.BREV}
                 onNesteSteg={validerOgLagreVedtak}
+                validerUlagedeKomponenter={false}
```

Comment:
Hvorfor skal denne være false?

Reply:
Den er default true, settes til false når man eks på vedtakssiden lagrer ned data via "gå til neste steg" og då ikke trenger å validere ulagret data.
Mens på inngangsvilkår så må man lagre data først, for å kunne gå til neste steg